compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -433,7 +439,8 @@ class Allocator {
   }
 
   NOINLINE void deallocate(void *Ptr, Chunk::Origin Origin, uptr DeleteSize = 0,
-                           UNUSED uptr Alignment = MinAlignment) {
+                           bool HasDeleteSize = false, uptr DeleteAlignment = 0,
+                           bool HasDeleteAlignment = false) {
```

Comment:
Can we use `DeleteAlignment=0` to indicate `HasDeleteSize=false`?

Reply:
This would prevent us from detecting `operator delete(<valid_ptr>, 0)` vs `operator delete(<valid_ptr>)`.